doc/release/1.8.0-notes.rst

```diff
@@ -50,80 +86,808 @@ New features
--------------------------
+- TransformedDensityRejection
+- DiscreteAliasUrn
+- NumericalInversePolynomial
+- DiscreteGuideTable
+- SimpleRatioUniforms
 
+The ``binned_statistic`` set of functions now have improved performance for
+the ``std``, ``min``, ``max``, and ``median`` statistic calculations.
 
-Other
------
+``somersd`` and ``_tau_b`` now have faster Pythran-based implementations.
 
+Some general efficiency improvements to handling of ``nan`` values in
+several ``stats`` functions.
 
+Added the Tukey-Kramer test as `scipy.stats.tukey_hsd`.
 
+Improved performance of `scipy.stats.argus.rvs`.
+
+Added the parameter ``keepdims`` to `scipy.stats.variation` and prevent the
+undesirable return of a masked array from the function in some cases.
 
 *******************
 Deprecated features
 *******************
 
-`scipy.linalg` deprecations
-===========================
+Namespaces that were private but happened to miss underscores have been
+deprecated. These include:
 
+- `scipy.signal.spline`
+- `scipy.ndimage.filters`
+- `scipy.ndimage.fourier`
+- `scipy.ndimage.measurements`
+- `scipy.ndimage.morphology`
+- `scipy.ndimage.interpolation`
 
-`scipy.spatial` deprecations
-============================
+All functions in these namespaces are accessible from their respective public
+namespace (e.g. `scipy.signal`). See
+`Issue 14360 <https://github.com/scipy/scipy/issues/14360>`_ for more details.
 
+``NumericalInverseHermite`` has been deprecated from `scipy.stats` and moved
+to the `scipy.stats.sampling` submodule. It now uses the C implementation of
+the UNU.RAN library so the result of methods like ``ppf`` may wary slightly.
+Parameter ``tol`` has been deprecated and renamed to ``u_resolution``. The
+parameter ``max_intervals`` has also been deprecated and will be removed in a
+future release of SciPy.
 
 
 ******************************
 Backwards incompatible changes
 ******************************
 
+- SciPy has raised the minimum compiler versions to GCC 6.3 on linux and
+  VS2019 on windows. In particular, this means that SciPy may now use C99 and
+  C++14 features. For more details see
+  `here <https://docs.scipy.org/doc/scipy/reference/dev/toolchain.html>`_.
+- The result for empty bins for `scipy.stats.binned_statistic` with the builtin
+  ``'std'`` metric is now ``nan``, for consistency with ``np.std``.
+- The function `scipy.spatial.distance.wminkowski` has been removed. To achieve
+  the same results as before, please use the ``minkowski`` distance function
+  with the (optional) ``w=`` keyword-argument for the given weight.
+
 *************
 Other changes
 *************
 
+Some Fortran 77 code was modernized to be compatible with NAG's nagfor Fortran
+compiler (see, e.g., `PR 13229 <https://github.com/scipy/scipy/pull/13229>`_).
 
+``threadpoolctl`` may now be used by our test suite to substantially improve
+the efficiency of parallel test suite runs.
 
 *******
 Authors
 *******
 
+* @endolith
+* [Sara Fridovich-Keil] +
```

Comment:
remove brackets from name. Looks like we need a whole bunch of mailmap updates.

Reply:
Ok, I went through and manually adjusted the `.mailmap` for every suspicious/malformed name I could find where some extra information was available. In some cases I even clicked through to user webpages to find actual names that were not on GitHub profiles.

I'll mark this resolved now.

Reply:
> for every suspicious/malformed name I could find

In case it helps, I found `git shortlog -se <version>..HEAD` very helpful for that. There are also some scripts out there to help cleanup mailmap, although mailmap has some surprises up its sleeve.